bftengine/tests/simpleKVBC/src/ReplicaImp.h

```diff
@@ -17,11 +17,21 @@
 #include <memory>
 
 #include "KVBCInterfaces.h"
-#include "BlockchainDBAdapter.h"
+#include "blockchain/db_interfaces.h"
+#include "blockchain/db_adapter.h"
 #include "SimpleBCStateTransfer.hpp"
 #include "Replica.hpp"
 #include "Metrics.hpp"
 
+using concord::storage::blockchain::ILocalKeyValueStorageReadOnly;

```

Comment:
`using` is strongly discouraged in header files, since they get injected into all files that include them. I guess this small test utility doesn't make a huge difference.

Reply:
+1 against "using".

Reply:
I removed them all.